api/allocations.go

```diff
@@ -112,6 +112,14 @@ type AllocationMetric struct {
 	Scores             map[string]float64
 	AllocationTime     time.Duration
 	CoalescedFailures  int
+	ScoreMetaData      map[string][]*NodeScoreMeta
+}
+
+// NodeScoreMeta is used to serialize node scoring metadata
+// displayed in the CLI during verbose mode
```

Comment:
Mark scores as deprecated here and in the website/api docs/changelog

Reply:
How do we mark fields in json responses as deprecated?